lib/facter/custom_facts/util/resolution.rb

```diff
@@ -139,7 +139,7 @@ def options(options)
       def setcode(string = nil, &block)
         if string
           @code = proc do
-            output = Facter::Core::Execution.execute(string, on_fail: nil)
+            output = Facter::Core::Execution.execute_command(string)
```

Comment:
Maybe this should be keep as `Facter::Core::Execution.execute(string, on_fail: nil)` as it expands commands.

Reply:
reverted the change